neo/Consensus/ConsensusService.cs

```diff
@@ -257,9 +257,12 @@ private void OnPerpareRequestReceived(ConsensusPayload payload, PerpareRequest m
             context.Signatures[payload.ValidatorIndex] = message.Signature;
             Dictionary<UInt256, Transaction> mempool = LocalNode.GetMemoryPool().ToDictionary(p => p.Hash);
             foreach (UInt256 hash in context.TransactionHashes.Skip(1))
-                if (mempool.ContainsKey(hash))
-                    if (!AddTransaction(mempool[hash], false))
+            {
+                Transaction tx;
```

Comment:
Thanks, I wasn't sure what MSBUILD is on the CI. I've converted all usages to inlined.